packages/react-core/src/demos/CardDemos.md

```diff
@@ -260,3 +253,163 @@ class CardGridDemo extends React.Component {
+          </CardBody>
+          <CardFooter>
+            <Button variant="tertiary">Call to action</Button>
+          </CardFooter>
+        </GridItem>
+      </Grid>
+    </Card>
+  );
+}
+```
+
+### Card details
+
+```js
+import React from 'react';
+import {
+  Card,
+  CardTitle,
+  CardBody,
+  CardFooter,
+  Gallery,
+  Title,
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+  Divider
+} from '@patternfly/react-core';
+
+CardDetailsDemo = () => {
+  return (
+    <Gallery hasGutter style={{'--pf-l-gallery--GridTemplateColumns--min': '260px'}}>
+      <Card>
+        <CardTitle>
+          <Title headingLevel="h2" size="xl">
+            Details
+          </Title> 
+        </CardTitle>
+        <CardBody>
+           <DescriptionList>
+            <DescriptionListGroup>
+              <DescriptionListTerm>Cluster API Address</DescriptionListTerm>
+              <DescriptionListDescription>
+                <a href="#">https://api1.devcluster.openshift.com</a>
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                Cluster ID
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                63b97ac1-b850-41d9-8820-239becde9e86
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>Provide</DescriptionListTerm>
+              <DescriptionListDescription>
+                AWS
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>OpenShift Version</DescriptionListTerm>
+              <DescriptionListDescription>
+                4.5.0.ci-2020-06-16-015028
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+            <DescriptionListGroup>
+              <DescriptionListTerm>Update Channel</DescriptionListTerm>
+              <DescriptionListDescription>
+                stable-4.5
+              </DescriptionListDescription>
+            </DescriptionListGroup>
+          </DescriptionList>
+        </CardBody>
+        <Divider />
+        <CardFooter>
+          <a href="#">View Settings</a>
+        </CardFooter>
+      </Card>
+      <Card>
+        <CardTitle>
+          <Title headingLevel="h2" size="xl">
+            Details
+          </Title>
+        </CardTitle>
+        <CardBody>
+          <DescriptionList>
+            <DescriptionListGroup>
+              <DescriptionListTerm>
+                Cluster API Address
+              </DescriptionListTerm>
+              <DescriptionListDescription>
+                https://api2.devcluster.openshift.com
```

Comment:
This feels a bit odd to see a link as plain text. Could we change this to a link, or change the description/descriptionListTerm to use less link-looking text?

Reply:
Yeah this seems like an oversight. I think we should link this text like the other card - @mcarrano do you agree?

Reply:
Yup.  I agree. it looks odd. This is how the core implementation looks.  @mnolting can you see @mcarrano's question above
https://github.com/patternfly/patternfly-react/pull/6193#pullrequestreview-734011794

Reply:
It looks like neither of the URLs are linked in the original design - https://marvelapp.com/prototype/eb14d81/screen/70405126

Reply:
Yes, Let's just make that a link for consistency.

Reply:
@mcoker Do you want to open a core issue for the update.